apps/api_web/lib/api_web/views/live_facility_view.ex

```diff
@@ -14,6 +14,7 @@ defmodule ApiWeb.LiveFacilityView do
 
   attributes([:properties, :updated_at])
 
+  def type(_, %{assigns: %{api_version: ver}}) when ver >= "2019-07-01", do: "live_facility"
```

Comment:
I don't have a very strong preference, but I find it a bit more readable when the case with the API version is for old versions and the default is for new versions, so I would slightly prefer `def type(_, _), do: "live_facility"` and then check if we're below 2019-07-01 to return `"live-facility"`.

Reply:
Good point, refactored.